### ISSUE TITLE
Trace ID and Span ID are not present in the output of DynamicSerilogCore

### ISSUE THREAD

**Issue by spanhotra** [SteeltoeOSS/Steeltoe#835]
## Describe the bug
A clear and concise description of the bug.  

## Steps to reproduce 
1. Create a sample application 
2. Added references of Steeltoe.Management.TracingCore and Steeltoe.Extensions.Logging.DynamicSerilogCore
3. In the Program.cs added following lines of code 
builder.Host.AddDynamicSerilog();
builder.Services.AddDistributedTracingAspNetCore();

## Expected behavior
In the console, I should receive the [appname, traceid, spanid]

## Environment (please complete the following information):
 - Steeltoe Version - 3.1.3
 - Platform: Local
 - OS: Windows Server 2019 
 - .NET Version .NET 5, .NET 6, .NET 7
 

## Screenshots
![image](https://user-images.githubusercontent.com/79205065/161965549-05acccf7-a39c-40cf-90cf-a8c0cc4db47f.png)

**Comment by spanhotra** [comment]
On further analysis. I believe the released version "3.1.3" is not same as the code base in this repo.
The structure of SerilogDynamicProvider in the version 3.1.3 is 
 
`public SerilogDynamicProvider(IConfiguration configuration, ISerilogOptions options = null, Logger logger = null, LoggingLevelSwitch loggingLevelSwitch = null);`

And in the code base is 

`public SerilogDynamicProvider(IOptionsMonitor<SerilogOptions> serilogOptionsMonitor, IEnumerable<IDynamicMessageProcessor> messageProcessors = null)
            : base(() => GetDelegateLogger(serilogOptionsMonitor), GetInitialLevelsFromOptions(serilogOptionsMonitor), messageProcessors)
        {
        }`

The message processors are not present, hence the information is not getting populated.

Please share the release path for these changes.

**Comment by hananiel** [comment]
@spanhotra, yes you are correct. This should be in 3.2.0 release - should be out in a week.

**Comment by spanhotra** [comment]
@hananiel Is this part of the latest release?

**Comment by spanhotra** [comment]
I see that it has been incorporated.
Thanks
